backend/apis/clients/follow-ups/create-follow-up.api.js

```diff
@@ -72,6 +76,15 @@ app.post("/api/clients/:clientId/follow-ups", (req, res) => {
       );
     });
 
+    query += insertActivityLogQuery({
+      detailId: insertResult.insertId,
+      clientId,
+      title,
+      description,
```

Comment:
Description should be `null`

 ```suggestion
      description: null,
```